control-plane/src/main.rs

```diff
@@ -75,14 +75,16 @@ async fn main() -> Result<()> {
             provisioner_proxy_result,
             acme_proxy_result,
             _,
+            enclave_boot_result,
         ) = tokio::join!(
             tcp_server(),
             e3_proxy.listen(),
             health_check_server.start(),
             config_server.listen(),
             provisioner_proxy.listen(),
             acme_proxy.listen(),
-            StatsProxy::listen()
+            StatsProxy::listen(),
+            Orchestration::start_enclave()
```

Comment:
Just need to add the same for when egress is enabled

Reply:
ahh yes good spot